app/src/map/components/Map.jsx

```diff
@@ -22,6 +22,11 @@ class Map extends React.Component {
     // there is a problem with the container width computation (only with "fat scrollbar" browser/os configs),
     // seems like the panels with scrollbars are taken into account or smth
     window.setTimeout(() => this._resize(), 10000);
+
+    // Shows the language selector hidden until map load finish
+    if (bablic !== undefined && bablic.widget && typeof bablic.widget.show === 'function') {
```

Comment:
I think this should be somewhere else, as `<Map />` is part of the map module, not the app where translations should happen

Reply:
Indeed, this needs to be moved when `as-module` branch goes to production, but for now, I thought it could fit here.